code/game/objects/items/storage/boxes.dm

```diff
@@ -845,6 +845,14 @@
 	else
 		return ..()
 
+/obj/item/storage/box/clown/suicide_act(mob/user)
+	user.visible_message(span_suicide("[user] opens [src] and gets consumed by [src.p_them()]! It looks like [user.p_theyre()] trying to commit suicide!"))
+	playsound(user, 'sound/misc/scary_horn.ogg', 70, TRUE)
+	var/obj/item/clothing/head/mob_holder/M = new(src, user)
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#use-descriptive-and-obvious-names